contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -141,10 +142,12 @@ def _eval_subtree(repo_url, subtree):
     @staticmethod
     def _eval_repo_url(repo_url):
         """Allow passing short GitHub style URLs"""
+        has_git_extension = repo_url.endswith('.git')
         if len(repo_url.split('/')) == 2 and not "git@" in repo_url:
-            return "https://github.com/{}.git".format(repo_url)
+            url = "https://github.com/{}".format(repo_url)
         else:
-            return "{}.git".format(repo_url)
+            url = repo_url
+        return url if has_git_extension.search(url) else "{}.git".format(url)
```

Comment:
`has_git_extension` will be `True` or `False` and at that point has_git_extension.search(url) works?

I have my doubts that this works. I will test and if broken will push changes.